### ISSUE TITLE
profile grows when adding trusted apps

### ISSUE THREAD

**Issue by michielbdejong** [SolidOS/solid-panes#63]
On gitter, @angelo-v [wrote](https://gitter.im/solid/node-solid-server?at=5c90f5a0f3dbbd230c808a23):
> Just tried the "trusted app" pane in NSS 5. My profile grows from 2.9K to 267K after doing so and contains tons of triples that should not be there... Anyone experienced this already?
Just seems that the whole knowledge graph has been PUT there by the client

**Comment by angelo-v** [comment]
Version: solid-server@5.0.0 from npm

## How to reproduce?

1. Go to your solid profile page (logged in)
2. Open the "Manage your trusted applications pane"
3. Enter an application URL, e.g. "https://markbook.org"
4. Select desired Access modes (I chose Read, Write, Append, but do not know if it is relevant)
5.  Click the add button

## Expected behaviour
A triple is added to the solid profile, stating that the app is trusted

## Actual behaviour
Countless, non-related and non-desired triples are added to the profile. (excerpt below). This leads to a approx. factor 10 growth of the file size of card$.ttl.

## Excerpt of unwanted triples

```
ind:fullNameField
    a ui:SingleLineTextField;
    ui:label "Name";
    ui:maxLength "128";
    ui:property n:fn;
    ui:size "40".
ind:fullNameFieldC
    a ui:SingleLineTextField;
    ui:maxLength "128";
    ui:property n:organization-name;
    ui:size "40".
ind:id1409437207443
    a ui:SingleLineTextField;
    ui:maxLength "128";
    ui:property n:street-address;
    ui:size "40".
ind:id1409437292400
    a ui:SingleLineTextField;
    ui:maxLength "128";
    ui:property n:locality;
    ui:size "40".
ind:id1409437421996
    a ui:SingleLineTextField;
    ui:maxLength "25";
    ui:property n:postal-code;
    ui:size "25".
ind:id1409437467649
    a ui:SingleLineTextField;
    ui:maxLength "128";
    ui:property n:region;
    ui:size "40".
ind:id1409437569420
    a ui:SingleLineTextField;
    ui:maxLength "128";
    ui:property n:country-name;
    ui:size "40".
ind:id1409437646712 a ui:Classifier; ui:from rdf:Class; ui:property terms:type.

ind:noteComment a ui:Comment; ui:contents "General Notes".
```

**Comment by bourgeoa** [comment]
NSS v5.0.0
And when you add more than one trusted app, permissions of the new one's are limited to 'read' 
All others permissions are not taken. You need to edit them manually.

**Comment by Vinnl** [comment]
I think this was solved by #79 - it no longer occurs for me.

**Comment by megoth** [comment]
Yes, this should be fixed, but we need to push the updates to NPM. (Should happen soon.)
